deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1861,30 +1828,24 @@ def registry_value_cud(root_key, registry_sub_key, log_monitor, arch=KEY_WOW64_6
             value_added_content = 0
             value_default_content = 1
 
-        if not isinstance(value_list, list) and not isinstance(value_list, dict):
-            raise ValueError('Value error. It can only be list or dict')
-        elif isinstance(value_list, list):
-            aux_dict = {registry_path: (value_default_content, callback_detect_event)}
-
+        aux_dict = {}
+        if isinstance(value_list, list):
             for elem in value_list:
                 aux_dict[elem] = (value_default_content, callback)
 
-            value_list = aux_dict
         elif isinstance(value_list, dict):
-            aux_dict = {registry_path: (value_default_content, callback_detect_event)}
-
             for key, elem in value_list.items():
                 aux_dict[key] = (elem, callback)
 
-            value_list = aux_dict
+        else:
+            raise ValueError('It can only be list or dict')
```

Comment:
```suggestion
            raise ValueError('It can only be a list or dictionary')
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1670/commits/c5f646dba669a1facacd74025ba515b4b452bda6)